apophis/__main__.py

```diff
@@ -20,15 +30,20 @@ def __init__(self, **kwargs):
         self.pool = kwargs.pop('pool')
 
     async def on_ready(self):
-        print('Logged in as ID: {}, username: {}'.format(self.user.id,
-                                                         self.user.name))
+        if DEBUG is True:
```

Comment:
We don't need to check whether DEBUG is true every time we want to issue a debug statement. If the log level in `logging` is set to a level higher than debug, logs made with `logging.debug` will be ignored.